### PR TITLE
569 - Edit community - losing about_us text

### PR DESCRIPTION
**Pull request by BradHN1**
There were 3 versions of the same form, I combined two of them to eliminate the bug;

### CODE REVIEW

**Comment by BradHN1** [comment]
There are no visual changes, but the problem where a couple fields didn't get updated in the form where rendered are fixed.

File: app/containers/MassEnergizeSuperAdmin/Community/EditCommunityByCommunityAdmin.js [hunk 1]
```diff
@@ -1,432 +1,41 @@
...
-    });
-  }
+class EditCommunityByCommunityAdmin extends React.Component {
 
   render() {
-    const { classes } = this.props;
-    const { formJson, community } = this.state;
-    if (!community || !formJson) return <Loading />;
+    const description = brand.desc;
+    console.log(this.props)
```
Comment (abdullai-t) [inline app/containers/MassEnergizeSuperAdmin/Community/EditCommunityByCommunityAdmin.js:14]: is there a reason for this log statement?

**Review by abdullai-t — APPROVED** [review]
Looks Good ✅

File: app/containers/MassEnergizeSuperAdmin/Community/EditCommunityByCommunityAdmin.js (hunk 1 above)
Comment (Davidjwkinghorn) [reply app/containers/MassEnergizeSuperAdmin/Community/EditCommunityByCommunityAdmin.js]: I'll remove it when I merge it. Good catch , BTW. I missed it when I started reviewing this on Monday.

**Review by Davidjwkinghorn — APPROVED** [review]
LGTM

**Pull request merged**